### PR TITLE
Adding animated hovers

### PR DESCRIPTION
**Pull request by PabloFLPs**
Adding some hovers to animated underline on navbar and Github/Twitter links and button.

### CODE REVIEW

File: src/styles/modules/_contact.module.scss [hunk 1]
```diff
@@ -6,4 +6,9 @@
   .send-message {
     margin: 0;
   }
+
+  button:hover {
+    filter: brightness(0.85);
+    transition: filter 0.5s;
+  }
```
Comment (Erickinhou) [inline src/styles/modules/_contact.module.scss:18]: Seria uma boa colocar o transition fora do hover pois aí eu tenho o efeito de fadeOut também. 
Do jeito que está agora o transition ele é meio que condicional ao hover.

```suggestion
    transition: filter 0.5s;
  button:hover {
    filter: brightness(0.85);
  }
```

File: src/styles/modules/_footer.module.scss [hunk 2]
```diff
@@ -20,7 +20,13 @@
...
       text-decoration: none;
       filter: invert(1);
-      padding: 0rem 0rem 0rem 1.5rem;
+      //padding: 0rem 0rem 0rem 1.5rem;
+      margin: 0 0 0 1.5rem;
+    }
+
+    a:hover {
+      box-shadow: 0 2px white;
+      transition: box-shadow 0.8s;
```
Comment (Erickinhou) [inline src/styles/modules/_footer.module.scss:32]: Same as above.

File: src/styles/modules/_navBar.module.scss [hunk 3]
```diff
@@ -10,15 +10,17 @@
...
     color: white;
     font-weight: 600;
+  }
 
-    &:hover {
-      text-decoration: underline;
-    }
+  a:hover {
+    box-shadow: 0 2px white;
+    transition: box-shadow 0.8s;
```
Comment (Erickinhou) [inline src/styles/modules/_navBar.module.scss:26]: same as above.

**Review by Erickinhou — APPROVED** [review]
Muito bom! Só alguns comentários.

File: src/styles/modules/_contact.module.scss (hunk 1 above)
Comment (PabloFLPs) [reply src/styles/modules/_contact.module.scss:18]: Oh, sure! Don't tried this, I was thinking in a way to make the animation on cursos out of the element. Thanks!

Comment (PabloFLPs) [reply src/styles/modules/_contact.module.scss:18]: Hi Erick, I tested the way that was suggested, but not woks as I thought. But I find a good alternative to do so:
```
button {
    filter: brightness(1);
    transition: filter 0.5s;
  }

button:hover {
  filter: brightness(0.85);
}
```

**Pull request merged**